packages/contracts-bedrock/src/L1/ETHLockbox.sol

```diff
@@ -85,24 +88,25 @@ contract ETHLockbox is ProxyAdminOwnedBase, Initializable, ISemver {
     }
 
     /// @notice Initializer.
-    /// @param _superchainConfig The address of the SuperchainConfig contract.
+    /// @param _systemConfig The address of the SystemConfig contract.
     /// @param _portals The addresses of the portals to authorize.
     function initialize(
-        ISuperchainConfig _superchainConfig,
+        ISystemConfig _systemConfig,
         IOptimismPortal[] calldata _portals
     )
         external
         initializer
     {
-        superchainConfig = ISuperchainConfig(_superchainConfig);
+        systemConfig = _systemConfig;
+        superchainConfig = systemConfig.superchainConfig();
```

Comment:
Let's get rid of the hard-coded SuperchainConfig reference - otherwise it becomes way harder to change the reference down the line if necessary.